test/auto_schedule/mat_mul.cpp

```diff
@@ -126,7 +126,7 @@ int main(int argc, char **argv) {
     std::cout << "======================" << std::endl;
 
     if (!get_jit_target_from_environment().has_gpu_feature() &&
-        (auto_time > manual_time * 3.5)) {
+        (auto_time > manual_time * 5.0)) {
```

Comment:
I think this change should be reverted, as it is unrelated to this PR. I think the autoschedule tests shouldn't be blocking this (or any) PR.

Reply:
Done. It might fail build/presubmit tests until https://github.com/halide/Halide/pull/4153 is submitted, but then it'll be fine.